db-cleaner/db-connection-env

```diff
@@ -0,0 +1,5 @@
+export PGHOST=db
```

Comment:
this should use this file https://github.com/GKernelCI/GKCI/blob/master/db.env
for don't have to keep two same files

Reply:
a minor hiccup with that is that `db.env` is not within the build context of db-cleaner,  and cronie doesn't pass environment variables through to the processes it runs as part of the crontab.

It's a solvable problem, just not immediately obvious.

Reply:
I added this commit:
https://github.com/GKernelCI/GKCI/pull/32/commits/0d6c1d4d26824dc68a9533837e676b0d0db03e07

it will make use of db.env and its environment variables, and set the psql client environment variables based on what is set in there.